build.rb

```diff
@@ -0,0 +1,19 @@
+require 'fileutils'
+
+
+
+case ARGV.first
+when "clean"
+  puts "> cleaning..."
+  FileUtils.rm_rf('assets/uswds')
+  FileUtils.rm_rf('_site')
```

Comment:
Removing `_site`  seems like a slightly different concern and is generally managed by Jekyll itself. Maybe remove it?

Reply:
I actually use "npm run clean" to start over and not have to call multiple commands in jekyll and js. Will keep for now.